xdebug.ini

```diff
@@ -242,7 +242,7 @@
 ;
 ; Only in Xdebug versions > 2.1
 ;
-; Type: string, Default value: 
```

Comment:
This is an auto generated file (by https://github.com/xdebug/xdebug.org/blob/master/html/docs/convert.php), so probably better not to touch it directly unless the conversion script is also fixed.